inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -0,0 +1,378 @@
+library(tidyverse)
+```
+It is a little tricker to load Bioconductor packages, as they are often not stored on the Comprehensive R Archive Network (CRAN) where most packages live. There is a package, however, that lives on CRAN and serves as an interface between CRAN and the Bioconductor packages.
+
+To load a Bioconductor package, you must first install and load the BiocManager package, like so.
+
+`install.packages("BiocManager")`
+`library("BiocManager)`
+
+You can then use the function `BiocManager::install()` to install a Bioconductor package. To install the Annotate package, we would execute the following code.
+
+`BiocManager::install("annotate")`
+
+
+## Working with data
+
+### Data description
+
+The data used throughout this module were collected as part of an on-going oceanographic time-series program in Saanich Inlet, a seasonally anoxic fjord on the East coast of Vancouver Island, British Columbia.
+
+The data that you will use in R are 16S rRNA amplicon profiles of microbial communities at several depths in Saanich Inlet from one time-point in this series (August 2012). These ~300 bp sequences were processed using [mothur](https://www.mothur.org/wiki/Main_Page) to yield 97% (approximately species-level) operational taxonomic units (OTUs). 
+
+`Saanich_OTU_metadata.csv` is a comma-delimited table of counts of four OTUs in each sample, normalized to 100,000 sequences per sample and the corresponding conditions of each sample (Depth, NO2, NO3 etc.). 
+
+For a brief introduction to these data, see Hallam SJ et al. 2017. Monitoring microbial responses to ocean deoxygenation in a model oxygen minimum zone. Sci Data 4: 170158 [doi:10.1038/sdata.2017.158](https://www.nature.com/articles/sdata2017158).
+
+### Loading tabular data
+
+Data tables can be loaded into R using the tidyverse `read_*` function.
+
+in your file browser, create a `data` directory in your project directory. Download the [`Saanich_OTU_metadata.csv`](https://github.com/EDUCE-UBC/educer/blob/master/data-raw/Saanich_OTU_metadata.csv) file and save it in your `data` directory.
+
+For example, we can load our Saanich data into R with `read_csv` for a comma-separated file and specify the arguments that describe our data as follows:
+
+- `file`: the name of the file you want to load
+- `col_names`: can take either the value `TRUE` or `FALSE` and tells R if the first row contains column names
+
+```{r, eval = FALSE}
+read_csv(file="data/Saanich_OTU_metadata.csv", col_names = TRUE)
+```
+
+```{r, echo = FALSE}
+OTU_metadata_table <- combined
+```
+
+### Save data in the environment
+
+Since we want to do more with our data after reading it in, we need to save it as a variable in R as we did previously with the `<-` operator. You can choose to name the object whatever you like, though this module assumes the names used below.
+
+```{r eval = FALSE}
+OTU_metadata_table <- read_csv(file="data/Saanich_OTU_metadata.csv", col_names = TRUE)
+```
+
+### Data exploration
```

Comment:
Perhaps a note for Stephan more than Cathy, but the following section is redundant with a section in the `data_wrangling_basic` module.

Reply:
Yes, Cathy will remove the section next time.